src/models/costproxy.h

```diff
@@ -39,3 +40,50 @@ class CostProxy : public QSortFilterProxyModel
         return CallerCalleeProxyDetail::match(this, item->symbol);
     }
 };
+
+template<typename Model>
+class CostProxyDiff : public CostProxy<Model>
```

Comment:
better name it DiffCostProxy instead? It's a diff between costs, not a Diff between CostProxies ?